adal/build.gradle

```diff
@@ -41,6 +41,13 @@ android {
         }
     }
 
+    sourceSets {
+        main {
+            java {
+                srcDir 'src/telemetry/java'
```

Comment:
oooops, i add the src/main in msal, you didn't catch it. :)

---
In reply to: [76157136](https://github.com/AzureAD/azure-activedirectory-library-for-android/pull/703#discussion_r76157136) [](ancestors = 76157136,76130688)